src/assignments/tech-jobs-mvc.rst

```diff
@@ -3,4 +3,510 @@
+   for this project. Fork the repository to create a copy under your own GitHub
+   account.
+#. Back in IntelliJ, close any open projects (after saving and committing your
+   work).
+#. From the welcome screen, click *Check out from Version Control*, select
+   *Git*.
+#. Choose your fork from the repository dropdown, select the parent directory
+   where you want to store your project, and hit *Clone*.
+#. In the screens that follow:
+
+   a. Choose *Create Project From Existing Sources* on the first pane.
+   b. Select *Auto Import* in the Gradle configuration pane.
+   c. Use the defaults on all other panes.
+
+#. Start up the application (via the Gradle pane, *Tasks > Application >
+   bootRun*), so you can refer to both the code as well as the running app
+   while we look at what’s in place already.
+
+.. admonition:: Note
+
+   You’ll spend a lot of time reading and understanding the code that’s provided
+   for you. This is OK! It’s an essential skill for a programmer to be
+   able to read and understand code written by other developers.
+
+1) Review the Code
+-------------------
+
+Carly created a Spring Boot application and filled in some features. She
+refactored ``JobData`` to generate an ArrayList of ``Job`` objects based on
+your TechJobs-OO work, and she added controllers and views for a Home, List and
+Search page. ``JobData`` now also builds ArrayLists for the ``Employer``,
+``Location``, ``PositionType``, and ``CoreCompetency`` objects.
+
+The Model
+^^^^^^^^^^
+
+.. admonition:: Note
+
+   When referring to Java class locations, we’ll usually omit the leading
+   path: ``src/main/java/org/launchcode/``.
+
+The "model" is contained in the ``JobData`` class, which is in the ``models``
+package. We put "model" in quotes, since this class isn’t a model in the
+typical, MVC/object-oriented sense (maybe a better name for this assignment
+would be *TechJobs VC*).
+
+The ``JobData`` class serves the same purpose as before---it reads data from
+the ``job_data.csv`` file and stores it in a format we can use. In this case,
+that format is an ArrayList of ``Job`` objects. Note that Carly changed the
+path to the ``job_data.csv`` file to store it in the ``src/main/resources``
+directory.
+
+You’ll use some of the static methods provided by ``JobData`` in your
+controller code. Since you’re already familiar with these, we’ll leave it to
+you to review their functionality as you go.
+
+The Controllers
+^^^^^^^^^^^^^^^
+
+Expand the ``controllers`` package, and you’ll see that you have three
+controllers already in place. Let’s look at these one at a time.
+
+The ``HomeController``
+~~~~~~~~~~~~~~~~~~~~~~~
+
+This class has only one handler method, ``index``, which displays the home page
+for the app. The controller renders the ``index.html`` template (in
+``src/main/resources/templates``).
+
+If you haven’t done so already, launch the app and examine the home page to see
+what it looks like.
```

Comment:
I would give them a reminder to immediately launch the app with the starter code and see what it looks like. Will the starter code even run? What should they see? It might be helpful for people to understand the starter code.

Reply:
The starter code does run on its own, and I reworded the last Getting Started step to emphasize launching the app.  I also added a screenshot of the home page.